.jenkins/pipeline.jenkins

```diff
@@ -0,0 +1,686 @@
+        }
+        stage('centos7') {
+          steps {
+            //buildJob(".build/docker/build-redhat.sh noboolean", "*.rpm")
+            sh 'echo "stage centos7"'
+          }
+        }
+       stage('stress') {
+         steps {
+           //sh 'cp -r /tmp/* /home/jenkins/agent/workspace/k8s-e2e'
+           testJob(".build/docker/run-tests.sh", "stress-test", 1)
+           sh 'echo "stage stress"'
+         }
+       }
+       stage('fqltool') {
+         steps {
+           testJob(".build/docker/run-tests.sh", "fqltool-test", 1)
+           sh 'echo "stage fqltool"'
+         }
+       }
+      // XXX – commented out for infracloud/cassandra-5.0 branch
+      //  stage('units') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "test", 8)
+      //    }
+      //  }
+      //  stage('long units') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "long-test", 8)
+      //    }
+      //  }
+      //  stage('burn') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "test-burn", 8)
+      //    }
+      //  }
+      //  stage('cdc') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "test-cdc", 8)
+      //    }
+      //  }
+      //  stage('compression') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "test-compression", 8)
+      //    }
+      //  }
+      //  stage('cqlsh') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "cqlsh-test", 1)
+      //    }
+      //  }
+      //  stage('jvm-dtest') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "jvm-dtest", 8)
+      //    }
+      //  }
+      //  stage('jvm-dtest-upgrade') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "jvm-dtest-upgrade", 8)
+      //    }
+      //  }
+      //  stage('dtest') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest", 64)
+      //    }
+      //  }
+      //  stage('dtest-large') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-large", 8)
+      //    }
+      //  }
+      //  stage('dtest-novnode') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-novnode", 64)
+      //    }
+      //  }
+      //  stage('dtest-offheap') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-offheap", 64)
+      //    }
+      //  }
+      //  stage('dtest-large-novnode') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-large-novnode", 8)
+      //    }
+      //  }
+      //  stage('dtest-upgrade') {
+      //    steps {
+      //      testJob(".build/docker/run-tests.sh", "dtest-upgrade", 64)
+      //    }
+      //  }
+     }
+    }
+    stage('Summary') {
+      steps {
+        sh 'echo "stage summary"'
+        dir("build") {
+          unstash name: "cassandra_11_stress-test"
```

Comment:
why are we unstashing in the summary ?

Reply:
It was something we were planning earlier, but now we have all the required files saved on the master, so this is no longer needed